packages/connector-aliyun-sms/src/index.ts

```diff
@@ -27,11 +28,30 @@ export default class AliyunSmsConnector implements SmsConnector {
     }
   };
 
-  public sendMessage: SmsSendMessageFunction = async (phone, type, { code }, config) => {
-    const smsConfig =
-      (config as AliyunSmsConfig | undefined) ?? (await this.getConfig(this.metadata.id));
+  public sendMessage: SmsSendMessageFunction = async (phone, type, { code }) => {
```

Comment:
- I think they should not share the same function type
- sendTestMessage can be in the class but the implementation is optional, or it can has a default implementation